contracts/id.tmy/src/id.tmy.cpp

```diff
@@ -115,18 +115,22 @@ namespace idtmy
       {
          throwError("TCON1000", "This people username is already taken");
       }
-      
-      //Set the resource limits for the new account
-      eosiobios::bios::resource_config_table _resource_config("eosio.bios"_n, "eosio.bios"_n.value);
-      auto config = _resource_config.get_or_create(get_self(), eosiobios::bios::resource_config());
-      config.total_cpu_weight_allocated = this->initial_cpu_weight_allocation;
-      config.total_net_weight_allocated = this->initial_net_weight_allocation;
-      _resource_config.set(config, get_self());
+      // If the account name exists, set resources and return
+      auto existing_person = _people.find(random_name.value);
+      if (existing_person != _people.end())
+      {
+         eosio::set_resource_limits(random_name, 0, this->initial_cpu_weight_allocation, this->initial_net_weight_allocation);
+         return;
+      }
 
-      int64_t myRAM, net, cpu;
-      eosio::get_resource_limits( random_name, myRAM, net, cpu );
+      //Set the resource limits for the new account
+      //uncomment this in task TODOS #77 
 
-      eosio::set_resource_limits(random_name, myRAM, this->initial_cpu_weight_allocation, this->initial_net_weight_allocation);
+      // eosiobios::bios::resource_config_table _resource_config("eosio.bios"_n, "eosio.bios"_n.value);
+      // auto config = _resource_config.get_or_create(get_self(), eosiobios::bios::resource_config());
```

Comment:
why commented? this should not be commented.

Reply:
because it says     message: 'assertion failure with message: cannot create objects in table of another contract',
 and i thought before launch we move id.tmy to eosio.bios then this error will resolve.

Reply:
please create a task for this and add to the sprint. link to this comment